docs/releasehistory.rst

```diff
@@ -8,6 +8,12 @@ Bugfixes
 --------
 - Bug in statistical inefficiency computation -- where self.max_n_iterations wasn't being used -- was fixed (`#577 <https://github.com/choderalab/openmmtools/pull/577>`_).
 - Bug in estimated performance in realtime yaml file -- fixed by iterating through all MCMC moves (`#578 <https://github.com/choderalab/openmmtools/pull/578>`_)
+- Explicit update and broadcast of thermodynamic states when running equilibration. Issue `#579 <https://github.com/choderalab/openmmtools/issues/579>`_ (`#587 <https://github.com/choderalab/openmmtools/pull/587>`_).
```

Comment:
This one-sentence description doesn't explain to the user what bug this would have caused, which is what all the other descriptions do. Can you be more explicit about what problematic behavior was corrected?

Reply:
For example, is the correct explanation "Bug in equilibration not doing anything when running with MPI -- fixed by explicitly broadcasting thermodynamics states after equilibration"

Reply:
This doesn't really solve a bug, since the `equilibrate()` function is not really called inside an MPI Context. In my opinion this is just for "completness" to match the changes made to `_mix_replicas()` in https://github.com/choderalab/openmmtools/pull/562 (this one did fix the bug with mixing replicas in an MPI context).